administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -0,0 +1,53 @@
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `asset_id` int DEFAULT 0,
+  `title` varchar(255) NOT NULL,
+  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `description` text NOT NULL,
+  `ordering` int NOT NULL DEFAULT 0,
+  `created` datetime NOT NULL,
+  `created_by` int NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL,
+  `modified_by` int NOT NULL DEFAULT 0,
+  `checked_out_time` datetime NOT NULL,
+  `checked_out` int NOT NULL DEFAULT 0,
+  `published` tinyint NOT NULL DEFAULT 0,
+  `state` tinyint NOT NULL DEFAULT '1',
+  PRIMARY KEY (`id`),
+  KEY `idx_asset_id` (`asset_id`),
+  KEY `idx_title` (`title`(191)),
+  KEY `idx_created` (`created`),
+  KEY `idx_created_by` (`created_by`),
+  KEY `idx_modified` (`modified`),
+  KEY `idx_modified_by` (`modified_by`),
+  KEY `idx_checked_out` (`checked_out`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+-- --------------------------------------------------------
+  
+--
+-- Table structure for table `#__guidedtour_steps`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `tour_id` int NOT NULL DEFAULT 0,
+  `title` varchar(255) NOT NULL,
+  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `published` tinyint NOT NULL DEFAULT 0,
+  `description` text NOT NULL,
+  `ordering` int NOT NULL DEFAULT 0,
+  `created` datetime NOT NULL,
+  `created_by` int unsigned NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL,
+  `modified_by` int unsigned NOT NULL DEFAULT 0,
+  `state` tinyint NOT NULL DEFAULT '1',
+  KEY `idx_tour` (`tour_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
+
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `manifest_cache`, `params`, `custom_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
+(0, 'com_guidedtours', 'component', 'com_guidedtours', '', 1, 1, 0, 1, '', '{}', '', 0, NULL, 0, 0);
```

Comment:
```suggestion
INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
(0, 'com_guidedtours', 'component', 'com_guidedtours', '', 1, 1, 0, 0, 1, '', '{}', '', 0, NULL, 0, 0);
```
The extension should not be protected so it is possible to disable it, but it should be locked so it is not possible to uninstall it.